packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1193,6 +1193,35 @@ oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
 oie.user.profile.secondary.email= Secondary email
+oie.user.profile.middleName = Middle name
+oie.user.profile.honorificPrefix = Honorific prefix
+oie.user.profile.honorificSuffix = Honorific suffix
+oie.user.profile.primaryEmail = Primary email
+oie.user.profile.title = Title
+oie.user.profile.displayName = Display name
+oie.user.profile.nickName = Nickname
+oie.user.profile.profileUrl = Profile URL
+oie.user.profile.secondEmail = Secondary email
+oie.user.profile.mobilePhone = Mobile phone
+oie.user.profile.primaryPhone = Primary phone
+oie.user.profile.streetAddress = Street address
+oie.user.profile.city = City
+oie.user.profile.state = State
+oie.user.profile.zipCode = ZIP Code
+oie.user.profile.postalCode = Postal code
+oie.user.profile.countryCode = Country code
+oie.user.profile.postalAddress = Postal address
+oie.user.profile.preferredLanguage = Preferred language
+oie.user.profile.locale = Locale
+oie.user.profile.timezone = Time zone
+oie.user.profile.userType = User type
+oie.user.profile.employeeNumber = Employee number
+oie.user.profile.costCenter = Cost center
+oie.user.profile.organization = Organization
```

Comment:
Similar comment here as below, but "Organization" has multiple definitions. Let's give an example via a comment to help our translators.